index.bs

```diff
@@ -233,6 +234,20 @@ spec: ecma-262; urlPrefix: http://tc39.github.io/ecma262/
                 The user agent *should not* ask the user for confirmation during the
                 [=prompt to unload=] step (and so the browsing context should be
                 [=discard a browsing context|discarded=]).
```

Comment:
I don't think closing the browsing context is what we want here. Even if we don't ask for confirmation, running beforeunload _after_ the navigation has already finished wouldn't be the right time. If we do want to run beforeunload, it should be before step 4 of activate-a-portal-browsing-context. Also step 3 of closing (remove browsingContext from the user interface) is redundant given step 6 of activation. It seems like we should replace closing with unload and discard. WDYT?